crates/ruff_cli/src/commands/run.rs

```diff
@@ -226,3 +226,82 @@ with the relevant file contents, the `pyproject.toml` settings, and the followin
         }
     }
 }
+
+#[cfg(test)]
+#[cfg(unix)] // Remove when adding a second test
```

Comment:
Should this be `TODO(konsti)`?

Reply:
no it's just a note for when someone wants to add another test case so they don't trip over this